### DISCUSSION TITLE
Posthoc Denoising

### DISCUSSION THREAD

**Discussion by TKisk** [DrCoffey/DeepSqueak#168] [Q&A]
Hello, 
I am using the new version of DS and recently when I either try to train a posthoc denoising or use a posthoc denoising I am getting red errors in the command window. If anyone has some advice or tips on how I can fix this to use these functions I would really appreciate the feedback. I attached some screenshots of the errors I get. If it's relevant I am trying to use it on Rat USV in the 50kHz range and used the Rat detector YOLO R1. Unfortunately, my files have a noise band that the detection keeps identifying and I would like to train it to recognize it only as noise. I also attached the image of the USVs and the noise band for reference.  Thanks in advance for any help!
![DS error when doing posthoc training](https://user-images.githubusercontent.com/104892294/166643531-7dd9e6da-0358-4b85-ac25-ba7481702f94.JPG)
![DS error when running posthoc denoising](https://user-images.githubusercontent.com/104892294/166643542-e3cfb79a-31bd-40bc-a293-44dc7abcf394.JPG)
![USV example image](https://user-images.githubusercontent.com/104892294/166644003-2539a1fc-915a-4598-aa30-4002bb91d1fa.JPG)
t

**Comment by DrCoffey** [comment]
Hey, the posthoc denoising issue is now fixed!
